biodatasets/seth_corpus/seth_corpus.py

```diff
@@ -0,0 +1,224 @@
+                    ],
+                    "attributes": [  # M or A lines in brat
+                        {
+                            "id": datasets.Value("string"),
+                            "type": datasets.Value("string"),
+                            "ref_id": datasets.Value("string"),
+                            "value": datasets.Value("string"),
+                        }
+                    ],
+                    "normalizations": [  # N lines in brat
+                        {
+                            "id": datasets.Value("string"),
+                            "type": datasets.Value("string"),
+                            "ref_id": datasets.Value("string"),
+                            "resource_name": datasets.Value("string"),
+                            "cuid": datasets.Value("string"),
+                            "text": datasets.Value(
+                                "string"
+                            ),
+                        }
+                    ],
+                },
+            )
+
+        elif self.config.schema == "bigbio_kb":
+            features = schemas.kb_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        urls = _URLS[self.config.schema]
+        data_dir = Path(dl_manager.download_and_extract(urls))
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": data_dir/"SETH-master"/"resources"/"SETH-corpus",
+                    "corpus_file": "corpus.txt",
+                    "split": "train",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath: Path, corpus_file: str, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+
+        if self.config.schema == "source":
+            with open(filepath/corpus_file, encoding='utf-8') as f:
+                contents = f.readlines()
+            for guid, content in enumerate(contents):
+                file_name, text = content.split("\t")
+                example = parsing.parse_brat_file(filepath/"annotations"/f"{file_name}.ann")
+                example["id"] = str(guid)
+                example["text"] = text
+                yield guid, example
+        elif self.config.schema == "bigbio_kb":
+            with open(filepath/corpus_file, encoding='utf-8') as f:
+                contents = f.readlines()
+            for guid, content in enumerate(contents):
+                file_name, text = content.split("\t")
+                example = parsing.parse_brat_file(filepath / "annotations" / f"{file_name}.ann")
+                print(example)
```

Comment:
could you please remove the print statetments?

Reply:
sorry, I forgot to remove them